woocommerce/woocommerce-ios
# Dynamic Dashboard: Top Performers card enhancements

Closes: #12667


## Description

This PR adds a few enhancements to the Top Performers card:
- Updated the time range bar when a custom range is selected to be consistent with the Performance card.
- Fixed the text alignment issue on the product title and subtitle.
- Reverted the order of items on the time range picker view following beta-testing feedback in [p1715110072534779/1715109882.889099-slack-C03L1NF1EA3].
- Removed redundant file `DashboardUI`.

## Testing instructions

- Log in to a store with revenues and enable the Top Performers card.
- Confirm that text alignment is correct for products with long titles (or adjust the font size to make the titles break into two lines).
- Select the time range menu and confirm that the order of items are correct following the design.
- Switch between predefined time ranges and custom time ranges to confirm that the time range bar is displayed correctly.

## Screenshots

<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/32b95b2b-b715-4df2-b02e-b2f33cecb290" width=320 />
<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/fb81e616-f12b-427a-968c-41ecc8db7e18" width=320 />


---
